### DISCUSSION TITLE
Ability to remove post code

### DISCUSSION THREAD

**Discussion by capArturo507** [woocommerce/woocommerce#49509] [Checkout Flow]
Is it possible to remove post code? in some countries most people does not know what that is, even the government does not have a map o post codes to physical location

**Comment by senadir** [comment]
Postcode can be removed via code for now, I suggest you do this on a country per country basis

```php
add_filter('woocommerce_get_country_locale', function( $locale ) {

	$locale['DZ']['postcode']['required'] = false;
	$locale['DZ']['postcode']['hidden'] = true;

	return $locale;
});
```
